storybook/stories/Layout/TopBackground.tsx

```diff
@@ -0,0 +1,12 @@
+import React from "react";
+import { storiesOf } from "@storybook/react-native";
+import { TopBackground } from "../../../src/components/Layout/TopBackground";
+import { AppMode } from "../../../src/context/config";
+
+storiesOf("Layout", module).add("TopBackgroundProduction", () => (
```

Comment:
Nest within `TopBackground`

Reply:
nesting stories in RNC is not available =>   https://github.com/storybookjs/react-native/issues/12.
we will discuss more on this.. how we going to do the  stories hierarchy

Reply:
**one of the possible way like this** 

![File (1)](https://user-images.githubusercontent.com/66373739/110910341-249db300-834c-11eb-9ab0-1c61302569ed.jpg)